_targets.R

```diff
@@ -133,9 +133,9 @@ list(
     # stplanr::route(l = od_to_route, route_fun = cyclestreets::journey, plan = "balanced")
     folder_name = paste0("outputdata/", parameters$date_routing)
     
-    routes_commute = get_routes(od_commute_subset,
+    routes_commute = get_routes(od = od_commute_subset,
                                 plans = parameters$plans, purpose = "commute",
-                                folder = folder_name, batch = TRUE, nrow_batch = 1000000,
+                                folder = folder_name, batch = TRUE, nrow_batch = 99999,
```

Comment:
Because after testing I discovered that CycleStreets will accept 99999 OD pairs but arbitrarily not 100k pairs. I'll forward the email exchange with @mvl22 .